<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Supplier</class>
 <widget class="QWidget" name="Supplier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1296</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Suppliers</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="filter_categories_label">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Filter Supplier: </string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="supplier_filter_input">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="suppliers_table">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>150</number>
            </attribute>
            <column>
             <property name="text">
              <string>Id</string>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="textAlignment">
              <set>AlignCenter</set>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="textAlignment">
              <set>AlignCenter</set>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Address</string>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="textAlignment">
              <set>AlignCenter</set>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Phone</string>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="textAlignment">
              <set>AlignCenter</set>
             </property>
            </column>
            <column>
             <property name="text">
              <string>City</string>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="textAlignment">
              <set>AlignCenter</set>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Remarks</string>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="textAlignment">
              <set>AlignCenter</set>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QPushButton" name="delete_supplier_button">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Delete / Hapus</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="edit_supplier_button">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Edit / Ubah</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="add_supplier_button">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Add / Tambah</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="1" column="0">
             <widget class="QLabel" name="supplier_name_label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Supplier Name : </string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="supplier_name_input">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit{
	border: 1px solid #d1d5db;      /* Light gray border */
    border-radius: 6px;             /* Rounded corners */
    outline: none;                  /* Remove default outline */
    color: #333;                    /* Text color */
    background-color: #fff;       /* Light gray background */
    
}

QLineEdit:focus {
    border: 1px solid #93c5fd;       /* Blue border on focus */
    
    background-color: #ffffff;       /* White background when focused */
}</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="supplier_address_label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Supplier Address : </string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="supplier_address_input">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit{
	border: 1px solid #d1d5db;      /* Light gray border */
    border-radius: 6px;             /* Rounded corners */
    outline: none;                  /* Remove default outline */
    color: #333;                    /* Text color */
    background-color: #fff;       /* Light gray background */
    
}

QLineEdit:focus {
    border: 1px solid #93c5fd;       /* Blue border on focus */
    
    background-color: #ffffff;       /* White background when focused */
}</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="supplier_city_label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>City : </string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="supplier_city_input">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit{
	border: 1px solid #d1d5db;      /* Light gray border */
    border-radius: 6px;             /* Rounded corners */
    outline: none;                  /* Remove default outline */
    color: #333;                    /* Text color */
    background-color: #fff;       /* Light gray background */
    
}

QLineEdit:focus {
    border: 1px solid #93c5fd;       /* Blue border on focus */
    
    background-color: #ffffff;       /* White background when focused */
}</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="supplier_phone_number_label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Phone Number : </string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="supplier_phone_number_input">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit{
	border: 1px solid #d1d5db;      /* Light gray border */
    border-radius: 6px;             /* Rounded corners */
    outline: none;                  /* Remove default outline */
    color: #333;                    /* Text color */
    background-color: #fff;       /* Light gray background */
    
}

QLineEdit:focus {
    border: 1px solid #93c5fd;       /* Blue border on focus */
    
    background-color: #ffffff;       /* White background when focused */
}</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="supplier_remarks_label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Remarks : </string>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="2">
             <widget class="QTextEdit" name="supplier_remarks_input"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="supplier_id_label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Supplier Id : </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="supplier_id_input">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit{
	border: 1px solid #d1d5db;      /* Light gray border */
    border-radius: 6px;             /* Rounded corners */
    outline: none;                  /* Remove default outline */
    color: #333;                    /* Text color */
    background-color: #fff;       /* Light gray background */
    
}

QLineEdit:focus {
    border: 1px solid #93c5fd;       /* Blue border on focus */
    
    background-color: #ffffff;       /* White background when focused */
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="clear_supplier_button">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Clear Data</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="submit_supplier_button">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Submit</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="close_supplier_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Close / Tutup</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
